pkg/time/time_skew_amd64.go

```diff
@@ -25,6 +25,10 @@ import (
 	"github.com/chaos-mesh/chaos-mesh/pkg/ptrace"
 )
 
+type CompositeInjector struct {
```

Comment:
LGTM.
Hi @ReigenDing , thanks for your contribution.
In code orgnization , I think the `CompositeInjector` is not the best choice and we do not need the interface `FakeClockInjector`.
Since the difference between gettimeofday & clockgettime is the fake image and they are all used to apply time-skew, I think split two function in just injecting fake image is better.
Another problem is interface `CompositeInjector` restrict the accessibility of data like `deltaSeconds` & `deltaNanoSeconds` and only provide Inject & Recovery functions. But I think we cannot assume other developer do not need it . (Actually I need it) I think you can just inject it one by one & recover it one by one.
Example :
```
type TimeSkew struct {
	deltaSeconds int64
	deltaNanoSeconds int64
	clockIDsMask uint64
	fakeImageClockGetTime *FakeImage
        fakeImageGetTimeOfDay *FakeImage
}

func (it *TimeSkew) Inject(pid int) error {
xxxxx
if fakeImageClockGetTime != nil {
inject fakeImageClockGetTime 
}
if fakeImageGetTimeOfDay != nil {
inject fakeImageGetTimeOfDay 
}
xxxxx
}
```

Reply:
I introduced the `FakeClockInjector` in #2570 because we're going to have multiple implementations for injecting TimeChaos, TimeStop, and TimeSkew for `gettimeofday`.

At that time, I think the minimum required methods for a time chaos implementation are `Inject(pid int)` and `Recover(pid int)`. We could initialize one instance of `FakeClockInjector` and inject/recover to any processes.

> Another problem is interface CompositeInjector restrict the accessibility of data like deltaSeconds & deltaNanoSeconds and only provide Inject & Recovery functions. 

`CompositeInjector` is only a wrapper for running multiple `FakeClockInjector` in a fail-fast pattern, it should not contain any other logic. You should resolve the logic in the "real" implementation of `FakeClockInjector`, not in `CompositeInjector`

Reply:
> Another problem is interface CompositeInjector restrict the accessibility of data like deltaSeconds & deltaNanoSeconds and only provide Inject & Recovery functions.

Sorry， I should have said `FakeClockInjector ` here.

Reply:
> You should resolve the logic in the "real" implementation of `FakeClockInjector`.
I think that means the abstraction leaks .

Reply:
> > You should resolve the logic in the "real" implementation of `FakeClockInjector`.
> 
> I think that means the abstraction leaks .

I do not get your point. I do not get why writing codes in its own implementation of `FakeClockInjector` would cause abstraction leak. 🙈

Reply:
I think the abstraction leak here is something like we must get the same type from interface that the type makes `FakeClockInjector`. This will force us to check the type . 
I know we have no choice in golang , because we only have interface. 
I think if there are not many abstract behaviors on the interface , we shouldn't add it  before there is the requirments.